WordPress-iOS-Shared/Core/WPTableViewSectionHeaderFooterView.m

```diff
@@ -0,0 +1,227 @@
+#import "WPTableViewSectionHeaderFooterView.h"
+#import "WPTableViewCell.h"
+#import "WPStyleGuide.h"
+#import "NSString+Util.h"
+
+
+
+@interface WPTableViewSectionHeaderFooterView ()
+@property (nonatomic, strong) UILabel *titleLabel;
+@end
+
+
+@implementation WPTableViewSectionHeaderFooterView
+
+- (instancetype)initWithStyle:(WPTableViewSectionStyle)style
+{
+    NSParameterAssert(style == WPTableViewSectionStyleHeader || style == WPTableViewSectionStyleFooter);
+    
+    self = [super init];
```

Comment:
You should invoke the designated initializer of your base class here: initWithReuseIdentifier. If you aren't make the reuse identifier available for this class you can invoke with nil.

On the same page you should override that default initializer to use this initializer with a default style set.